include/mbedtls/config_psa.h

```diff
@@ -31,9 +31,17 @@
 #define MBEDTLS_CONFIG_PSA_H
 
 #if defined(MBEDTLS_PSA_CRYPTO_CONFIG)
+#if defined(MBEDTLS_PSA_CRYPTO_CONFIG_FILE)
+#include MBEDTLS_PSA_CRYPTO_CONFIG_FILE
+#else
 #include "psa/crypto_config.h"
+#endif
```

Comment:
We don't systematically do that, and we're in fact considering stopping doing it because the endif comments tend to bitrot. Given that the inner block is 5 lines long, we would normally not put an endif comment.